meson.build

```diff
@@ -364,6 +362,11 @@ if features['win32-internal-pthreads']
     # Hack around it by using HAVE_WIN32_INTERNAL_PTHREADS.
     includedir += include_directories('osdep/win32/include')
     sources += files('osdep/win32/pthread.c')
+else
+    # pthreads is intentionally left undefined in win32 branch to find incorrect
+    # uses of it immediately
+    pthreads = dependency('threads')
+    dependencies += pthreads
```

Comment:
Does this eliminate the need for the workaround described above (putting `HAVE_WIN32_INTERNAL_PTHREADS`in `common/stats.c`)?

Reply:
Not really, because apparently it is pulled from, `pthread_time.h` which is got pulled from `time.h`. But to make it clearer we can defined `#define _POSIX_TIMERS 0` in our wrapper header.

I don't like how we use the same name for a header `pthread.h` instead of adding `mp_thread.h` or something and include what's needed. But well it works as is.

Reply:
Feel free to change it if you want. When I added meson support I didn't want to mess with existing files too much since waf existed, but that's no concern anymore.